src/components/ChatEntry.js

```diff
@@ -1,22 +1,48 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const onLikeButtonClick = () => {
+    const newChatData = {
+      key: props.id,
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.updateChatLike(newChatData);
+  };
```

Comment:
This works just fine, and it's an example we show in Learn. 

But consider that `updateLiked` has been given the responsibility to update the likes, right? Let `updateLiked` take care of it by reassigning `liked`. All `onLikeButtonClick` should do is grab the `id` that `updateChatLike` needs and pass it through.